docs/reference/filebeat/filebeat-input-azure-blob-storage.md

```diff
@@ -336,6 +336,86 @@ filebeat.inputs:
 
 The Azure Blob Storage APIs don’t provide a direct way to filter files based on timestamp, so the input will download all the files and then filter them based on the timestamp. This can cause a bottleneck in processing if the number of files are very high. It is recommended to use this attribute only when the number of files are limited or ample resources are available.
 
+## `custom_properties` [attrib-custom-properties]
```

Comment:
ISTM that this should not be `custom_properties`; there is no field or attribute of that name, which is what this formatting suggests exists.

Reply:
Can you not just remove the code quotes and underscore, and then capitalise custom?

Reply:
yea sure I can do that but it will look out of place in the parent list: 
```
1. [account_name](#attrib-account-name)
2. [auth.oauth2](#attrib-auth-oauth2)
3. [auth.shared_credentials.account_key](#attrib-auth-shared-account-key)
4. [auth.connection_string.uri](#attrib-auth-connection-string)
5. [storage_url](#attrib-storage-url)
6. [containers](#attrib-containers)
7. [name](#attrib-container-name)
8. [max_workers](#attrib-max_workers)
9. [poll](#attrib-poll)
10. [poll_interval](#attrib-poll_interval)
11. [file_selectors](#attrib-file_selectors)
12. [expand_event_list_from_field](#attrib-expand_event_list_from_field)
13. [timestamp_epoch](#attrib-timestamp_epoch)
14. [Custom  properties](#attrib-custom-properties)
```

Reply:
Agree with @efd6 here. Let's remove backticks and format as "Custom properties"